tox.ini

```diff
@@ -69,6 +69,7 @@ commands =
 skip_install = True
 sitepackages = True
 deps = gcovr
+setenv = GCOV=arm-poky-linux-gnueabi-gcov
```

Comment:
I don't think this will work when testing locally ? or maybe I'm missing something

Reply:
You're right.